torch/_export/serde/schema.py

```diff
@@ -187,6 +187,26 @@ class Node:
     outputs: List[Argument]
     metadata: Dict[str, str]
 
+@dataclass
+class CoLineInfo:
+    name: str
+    line: str
+
+@dataclass
+class CoFileInfo:
+    file_name: str
+    line_info_map: Dict[str, CoLineInfo] # lineno is str in traceback.FrameSummary
+
+@dataclass
+class Frame:
+    filename: str
+    lineno: int
+    name: str
+    context: str
+
+@dataclass
+class FrameList:
+    items: List[Frame]
```

Comment:
These data types seem to be helper structures belonging to serialize.py, better to move them there.